module/zfs/dmu.c

```diff
@@ -153,6 +161,34 @@ const dmu_object_byteswap_info_t dmu_ot_byteswap[DMU_BSWAP_NUMFUNCS] = {
 	{	zfs_acl_byteswap,	"acl"		}
 };
 
+/*
```

Comment:
When you add new functions to dmu.c would it be possible to add them at the end? It wouldmake it a lot more pleasant resolving changes with the async dmu work.

Reply:
We wound up moving everything over to dmu_direct.c to help out with this.